app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PostsController < ApplicationController
```

Comment:
- [ ] Kindly consider using cancan authorization in the `Post controller` by adding `load_and_authorize_resource` to the controller